### PR TITLE
updating license for 2019

### PR DESCRIPTION
**Pull request by sslattery**

### CODE REVIEW

**Comment by sslattery** [comment]
will rebase on #57 for after it is merged so new tutorials also get new license years.

**Comment by sslattery** [comment]
@junghans Just thought it would be consistent if they all changed. Do most leave the original date the file was made?

**Comment by sslattery** [comment]
ping

**Comment by dalg24** [comment]
No opinion on this.  Did you and @junghans come to an agreement?

File: cmake/Cabana.pc.cmakein [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 # Use this in a Makefile:
 # myapp: myapp.cpp
-# 	$(CC) `pkg-config --cflags --libs Cabana` -o $@ $<
+#   $(CC) `pkg-config --cflags --libs Cabana` -o $@ $<
```
Comment (junghans) [inline cmake/Cabana.pc.cmakein]: A tab here was actually correct!

**Comment by sslattery** [comment]
Good idea on the script - let's implement when we get to next year

File: cmake/Cabana.pc.cmakein (hunk 1 above)
Comment (sslattery) [reply cmake/Cabana.pc.cmakein]: Does this affect the correctness of this file? This looks like my editor converted tabs to spaces or something.

Comment (junghans) [reply cmake/Cabana.pc.cmakein]: This a makefile rule, which needs a tab there, having spaces is just very error-prone. We could put `<TAB>` there to make it obvious if that would make it easier.

Comment (sslattery) [reply cmake/Cabana.pc.cmakein]: would this tab be to avoid copy and paste errors? I just now realized that was in a comment

**Pull request merged**